plugins/grpc/web/source.yaml

```diff
@@ -1,3 +1,7 @@
 source:
+  ignore_versions:
+    # Ignored until release assets are published.
+    # Ref: https://github.com/grpc/grpc-web/issues/1495
+    - v2.0.0
```

Comment:
I don't recall the syntax, should this have a leading v? (is it the raw string from upstream).